src/modules/commander/PreflightCheck.cpp

```diff
@@ -518,7 +518,7 @@ static bool ekf2Check(orb_advert_t *mavlink_log_pub, vehicle_status_s &vehicle_s
 	// Check if preflight check performed by estimator has failed
 	if (status.pre_flt_fail) {
 		if (report_fail) {
-			mavlink_log_critical(mavlink_log_pub, "PREFLIGHT FAIL: EKF INTERNAL CHECKS");
+			mavlink_log_critical(mavlink_log_pub, "Preflight Fail: EKF internal checks");
```

Comment:
what does this even mean?  Can this message be written more specifically and in a way to be understood by the average user?

Reply:
It's one that I would like to get rid of as well. @priseborough ?

Reply:
Let's remove or re-write the language.

Reply:
"Preflight fail: Position unknown" or something like that.

Reply:
I changed it to Position unknown.